theme/static/js/create-resource.js

```diff
@@ -21,95 +143,132 @@ $(document).ready(function() {
             $("#file-move-warning").hide();
         }
     });
-    $('#resource-type').on('change', function(){
-        $('#select-file').value = '';
-        $('#select-file').attr('value', '');
-        $('#irods-sel-file').text("No file selected.");
```

Comment:
@hyi, I have now included this behavior on resource type change. Please have a look and verify if it works correctly.